.travis.yml

```diff
@@ -4,9 +4,18 @@ branches:
       - /^v.*/
 android:
   components:
-  - build-tools-22.0.1
-  - android-22
+    # some travis/android magic from https://docs.travis-ci.com/user/languages/android/
+    - tools
+    - platform-tools
+    - tools
```

Comment:
Surely you don't mean to have `tools` twice? There is nothing at that URL that implies you need to do that, unless I am missing it?

Reply:
Finally, I've got rid of both `tools` references _and_ the `platform-tools`.